examples/nextjs-tiptap-ai/app/api/contextual-prompt/route.ts

```diff
@@ -0,0 +1,75 @@
+import { NextRequest, NextResponse } from "next/server";
+import { generateObject } from "ai";
+import { anthropic } from "@ai-sdk/anthropic";
+import * as z from "zod";
```

Comment:
💔

Reply:
Hopefully they'll accept the standard soon and we'll swap 😁

Reply:
❤️